pkgs/misc/themes/adapta/default.nix

```diff
@@ -1,14 +1,63 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, parallel, sassc, inkscape, libxml2, glib, gdk_pixbuf, librsvg, gtk-engine-murrine, gnome3 }:
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig, parallel, sassc, inkscape
+, libxml2, glib, gdk_pixbuf, librsvg, gtk-engine-murrine, gnome3
 
-stdenv.mkDerivation rec {
+, # Enable Gnome Shell support
```

Comment:
I think the comma should be in the same line as the flags. Most of these comments are pretty redundant anyways. I also don't much like the empty lines between all flags, but thats a matter of taste.

Reply:
I can just remove these.

Reply:
BTW, I based this off of existing arg documentation, such as `fetchurl`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/build-support/fetchurl/default.nix#L41

I agree that it looks odd.

Reply:
Yeah there is a lot of weird formatting in the repo and since we don't have a comprehensive style guide I can only review based on my personal preference. I try not to block merging on that though.

We really should have a style guide, I just don't want to be the one that leads the bikeshed.

Reply:
Some of the later comments are useful of course - just the `enableFoo # enable foo` ones could be removed.